Std/Tactic/Ext.lean

```diff
@@ -98,23 +109,41 @@ def applyExtLemma (goal : MVarId) : MetaM (List MVarId) := goal.withContext do
         let c ← mkConstWithFreshMVarLevels lem.declName
         let (_, _, declTy) ← withDefault <| forallMetaTelescopeReducing (← inferType c)
         guard (← isDefEq tgt declTy)
+      if tactic.ext.trace.get (← getOptions) then
+        let cmd ←`(tactic| apply $(mkIdent (← unresolveNameGlobal lem.declName)))
+        usedExtTactics.modify fun x => x.push cmd
       return ← goal.apply (← mkConstWithFreshMVarLevels lem.declName)
     catch _ => s.restore
   throwError "no applicable extensionality lemma found for{indentExpr ty}"
 
 /-- Apply a single extensionality lemma to the current goal. -/
-elab "apply_ext_lemma" : tactic => liftMetaTactic applyExtLemma
+elab "apply_ext_lemma" : tactic => do
+  usedExtTactics.modify fun _ => #[]
+  liftMetaTactic applyExtLemma
+  if tactic.ext.trace.get (← getOptions) then
+    let x ← usedExtTactics.get
+    let cmd ←`(tactic| · $x*)
+    logInfo m!"Try this:\n{cmd}"
```

Comment:
Is there a way to just print a sequence of tactics without any additional syntax?`` `(tactic| · $x*)`` adds the `·` and `` `(tactic| ($x*))`` adds parentheses around the block.